tests/unit/xoopsLib/Xoops/Core/XoopsTplTest.php

```diff
@@ -22,6 +24,7 @@ class XoopsTplTest extends \PHPUnit_Framework_TestCase
     protected function setUp()
     {
         $this->object = new XoopsTpl();
+        $this->xoops = \Xoops::getInstance();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Xoops' in method 'setUp'.](https://www.codacy.com/app/richard_3/XoopsCore/file/2444115150/issues/source?bid=3194562&fileBranchId=3245574#l27)